Cargo.toml

```diff
@@ -93,9 +93,16 @@ tracing = ["dep:tracing"]
 # internal features used in CI
 nightly = []
 
+[lints.rust.unexpected_cfgs]
+level = "warn"
+check-cfg = [
+    'cfg(hyper_unstable_tracing)',
+    'cfg(hyper_unstable_ffi)'
+]
+
 [package.metadata.docs.rs]
 features = ["ffi", "full", "tracing"]
-rustdoc-args = ["--cfg", "docsrs", "--cfg", "hyper_unstable_ffi", "--cfg", "hyper_unstable_tracing"]
+rustdoc-args = ["--cfg", "hyper_unstable_ffi", "--cfg", "hyper_unstable_tracing"]
```

Comment:
Is this a related change?

Reply:
Yes, I'm trying to make sure that all custom cfgs are declared, and `docsrs` is no longer a custom cfg since https://github.com/rust-lang/docs.rs/pull/2390. So, for consistency with above I removed it.

Reply:
Ok. I understand what you are addressing, but I would think it should be a separated pull request.